arrows/colmap/colmap.h

```diff
@@ -0,0 +1,33 @@
+#ifndef KWIVER_ARROWS_COLMAP_COLMAP_H
+#define KWIVER_ARROWS_COLMAP_COLMAP_H
+
+#include <arrows/colmap/kwiver_algo_colmap_export.h>
+#include <colmap/mvs/image.h>
+#include <iostream>
+#include <vital/types/image_container.h>
+
+namespace kwiver {
+
+namespace arrows {
+
+namespace colmap_arrow {
+
+KWIVER_ALGO_COLMAP_EXPORT
+void version();
```

Comment:
Let's remove this method since we put it in there just test we where correctly accessing the library.